Tests/scripts/configure_tests.py

```diff
@@ -975,7 +974,8 @@ def get_test_list(files_string, branch_name, two_before_ga_ver='0', conf=None, i
         tests = tests.union(get_test_from_conf(branch_name, conf))
 
     if not tests:
-        tests = get_random_tests(tests_num=RANDOM_TESTS_NUM, conf=conf, id_set=id_set, server_version=two_before_ga_ver)
+        rand = random.Random(files_string + branch_name)
```

Comment:
why not only branch name?

Reply:
I also think it is better just the branch name.

Reply:
It was like that before, but I'll switch to just branch name
```suggestion
        rand = random.Random(branch_name)
```